packages/progress-indicator/src/ProgressIndicatorHeading.tsx

```diff
@@ -0,0 +1,32 @@
+import { Stack } from '@ag.ds-next/box';
+import { Text } from '@ag.ds-next/text';
+import { tokens } from '@ag.ds-next/core';
+
+export const ProgressIndicatorHeading = ({
+	stepsCompleted,
+	totalSteps,
+}: {
+	stepsCompleted: number;
+	totalSteps: number;
+}) => {
+	return (
+		<Stack
+			gap={0.5}
+			paddingBottom={1.5}
+			css={{
+				display: 'none',
+				[tokens.mediaQuery.min.md]: {
+					display: 'flex',
+				},
+			}}
+		>
+			<Text fontSize="md" fontWeight="bold" lineHeight="heading">
```

Comment:
This should be a `h3`

Reply:
I really struggled with this. Not certain it should be a H3 or a H2. Should it be user configurable?

Reply:
Going to use a H2